packages/admin/cms-admin/src/warnings/warningConfig.ts

```diff
@@ -0,0 +1,14 @@
+import { type ReactNode } from "react";
+
+import { useCometConfig } from "../config/CometConfigContext";
+import { warningMessages as cometWarningMessages } from "./warningMessages";
+
+export interface WarningConfig {
```

Comment:
This should be WarningsConfig to be in line with the other configs.

Reply:
changed here: da0e362997fe1df2204f59c5734c5b1f42f9c314